2023/opmodes/TwoStickTeleop.java

```diff
@@ -0,0 +1,241 @@
+
+    private DcMotor liftMotor;
+    // private DcMotor intakeMotor;
+
+    private Servo coneServo;
+    private Servo junctionAligner;
+    private Servo armServo;
+
+    private boolean currentlySlow = false;
+
+    //private DeadmanMotorTask liftMotorUpTask;
+    //private DeadmanMotorTask liftMotorDownTask;
+    private OneWheelDriveTask liftMotorTask;
+
+//    private DeadmanMotorTask intakeTask;
+//    private DeadmanMotorTask outtakeTask;
+
+    MecanumFieldCentricDriveScheme scheme;
+
+    private MechanumGearedDrivetrain drivetrain;
+
+    private static final int TICKS_PER_INCH = 79;
+
+    @Override
+    public void handleEvent(RobotEvent e) {
+    }
+
+    @Override
+    public void init() {
+
+        super.init();
+
+        //mechanisms
+//        carouselMech = hardwareMap.get(DcMotor.class, "carouselMech");
+        liftMotor = hardwareMap.get(DcMotor.class,"liftMotor");
+//        intakeMotor = hardwareMap.get(DcMotor.class, "intakeMotor");
+
+        coneServo = hardwareMap.servo.get("coneServo");
+        junctionAligner = hardwareMap.servo.get("junctionAligner");
+        armServo = hardwareMap.servo.get("armServo");
+
+        // using encoders to record ticks
+        backLeft.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        backRight.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        frontLeft.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        frontRight.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+
+        coneServo.setPosition(0.4);
+        junctionAligner.setPosition(.2);
+        armServo.setPosition(0.8);
+
+        liftMotor.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
+
+        //telemetry
+        buttonTlm = telemetry.addData("buttonState", "unknown");
+
+        TwoStickMechanumControlScheme scheme = new TwoStickMechanumControlScheme(gamepad1);
+        drivetrain = new MechanumGearedDrivetrain(motorMap);
+        drivetrain.setNoncanonicalMotorDirection();
+        // Note we are swapping the rights and lefts in the arguments below
+        // since the gamesticks were switched for some reason and we need to do
+        // more investigation
+        drivetask = new TeleopDriveTask(this, scheme, frontLeft, frontRight, backLeft, backRight);
+
+        //liftMotorUpTask = new DeadmanMotorTask(this, liftMotor,  -1.0, GamepadTask.GamepadNumber.GAMEPAD_2, DeadmanMotorTask.DeadmanButton.LEFT_STICK_DOWN);
+       // liftMotorDownTask    = new DeadmanMotorTask(this, liftMotor, 1.0, GamepadTask.GamepadNumber.GAMEPAD_2, DeadmanMotorTask.DeadmanButton.LEFT_STICK_UP);
+        liftMotorTask = new OneWheelDriveTask(this, liftMotor, true);
+        liftMotorTask.slowDown(false);
+//        intakeTask = new DeadmanMotorTask(this, intakeMotor,  -0.5, GamepadTask.GamepadNumber.GAMEPAD_2, DeadmanMotorTask.DeadmanButton.RIGHT_STICK_UP);
+//        outtakeTask    = new DeadmanMotorTask(this, intakeMotor, 0.5, GamepadTask.GamepadNumber.GAMEPAD_2, DeadmanMotorTask.DeadmanButton.RIGHT_STICK_DOWN);
```

Comment:
Go ahead and remote all the commented out code.  If it's not applicable to this opmode it can be safely deleted entirely.